torch/csrc/jit/pickler.cpp

```diff
@@ -723,6 +729,48 @@ OpCode Unpickler::readInstruction() {
               AT_ERROR("Unknown pickler class id");
           }
         });
+      } else if (
+          module_name == "torch._utils" && class_name == "_rebuild_tensor_v2") {
+        globals_.emplace_back([this] {
+          auto tup = pop(stack_).toTuple();
+          const auto& elements = tup->elements();
+          auto storage_tensor = elements.at(0).toTensor();
+          int64_t storage_offset = elements.at(1).toInt();
+          std::vector<int64_t> size = tupleToIntList(elements.at(2));
+          std::vector<int64_t> stride = tupleToIntList(elements.at(3));
+          bool requires_grad = elements.at(4).toBool();
+          // elements[5] is empty backwards hooks
+          at::Tensor result =
+              at::empty({0}, storage_tensor.options())
+                  .set_(storage_tensor.storage(), storage_offset, size, stride);
+          result = autograd::make_variable(result, requires_grad);
+          stack_.push_back(std::move(result));
+        });
+      } else if (module_name == "collections" && class_name == "OrderedDict") {
```

Comment:
this is a bit scary. Is there are stricter condition we can enforce here (e.g. that it's indeed in Tensor deserialization only)